client/src/Components/Layouts/Captains/EditCaptain/editCaptain.jsx

```diff
@@ -0,0 +1,301 @@
+                })(<Input type="text" id="licenceNumber" />)}
+              </Form.Item>
+              <Form.Item label="الحالة">
+                {getFieldDecorator("status", {
+                  rules: [
+                    { required: true, message: "يرجى اختيار حالة" }
+                  ]
+                })(
+                  <Select style={{ width: 80 }} id='status'>
+                    <Option value="true">فعال </Option>
+                    <Option value="false">غير فعال</Option>
+                  </Select>
+                )}
+              </Form.Item>
+              <Form.Item label="صورة الهوية">
+                {getFieldDecorator("file", {
+                  rules: [
+                    {
+                      required: true,
+                      message: "يرجى رفع صورة الكابتن"
+                    }
+                  ]
+                })(<Upload
+                  accept=".jpg , .png, .jpeg"
+                  name="file"
+                  className="avatar-uploader"
+                  showUploadList={false}
+                  customRequest={this.handleReuest}
+                  multiple={false}
+                >
+                  {uploadButton}
+                  <div className="image__name">
+                    {(imageUrl) ? imageUrl && <><Icon type="check-circle" />{imageUrl.name}</> : photoUrl && <><Icon /><img src={photoUrl} className="upload-photo" /></>}
+
+                  </div>
+                </Upload>)}
+              </Form.Item>
+
+            </div>
+            <div className='edit-captain-container-center'></div>
+            <div className='edit-captain-container-right'>
+              <Form.Item label="الاسم">
+                {getFieldDecorator("name", {
+                  rules: [
+                    { required: true, message: "يرجى ملئ الحقل بحروف ", pattern: /^([أ-يa-z]|\s)+$/ }
+                  ]
+                })(<Input type="text" id="name"
+                />)}
+              </Form.Item>
+              <Form.Item label="البريد">
+                {getFieldDecorator("email", {
+                  rules: [
+                    {
+                      required: true,
+                      message: "يرجى ملئ الحقل بريد الكتروني",
+                      pattern: /.+\@.+\..+/
+                    }
+                  ]
+                })(<Input type="email" id="email" />)}
+              </Form.Item>
+              <Form.Item label="كلمة المرورالجديدة">
+                {getFieldDecorator("password", {
+                  rules: [
+                    {
+                    }
+                  ]
+                })(<Input type="password" id="password" />)}
+              </Form.Item>
+              <Form.Item label="الهاتف">
+                {getFieldDecorator("phone", {
+                  rules: [
+                    {
+                      required: true,
+                      message: "الرجاء ملئ الحقل بارقام",
+                      pattern: /^\+?[0-9]{10,12}$/
```

Comment:
@naremanhilles it might make your users' life easier if you allow dashes, spaces, and dots in the phone numbers too, so they can enter things like this:
+1 123-456-7890
+1 1234 56 789
+33 123.456.7890